cli/internal/cmd/buffer.go

```diff
@@ -402,6 +414,7 @@ func NewBufferWriteCommand(openFileFunc func(name string, flag int, perm fs.File
 	cmd.Flags().StringVarP(&inputFilePath, "input", "i", inputFilePath, "The file to read from. If not specified, data is read from standard in.")
 	cmd.Flags().IntVarP(&dop, "dop", "p", dop, "The degree of parallelism")
 	cmd.Flags().StringVarP(&blockSizeString, "block-size", "b", blockSizeString, "Split the stream into blocks of this size.")
+	cmd.Flags().StringVarP(&timeWindow, "time-window", "t", timeWindow, "Split the stream into blocks based on this time interval.")
```

Comment:
I think this should be a string that we call `time.ParseDuration()` on, so it can be `1s` or `500ms` etc. 
I think we could consider calling it `flush-interval`. The description could be something along the lines of:

> The longest time to wait before accumulated data is written to the remote service. If this parameter is set, data will be flushed either when `--block-size` of data has been accumulated or when the specified interval has elapsed, whichever comes first. This is useful for scenarios where data is trickling in and needs to be sent at regular intervals.

Reply:
I havent changed this yet, figured Id figure out buffers first - but will so this next.